helper_functions.R

```diff
@@ -408,12 +408,13 @@ sys_total_count_display <- function(total_count) {
       ),
       case_when(
         input$syso_hh_type == "All" ~ "",
-        str_detect(getNameByValue(syso_hh_types, input$syso_hh_type), "Household") == FALSE ~
-          paste(
-            " in",
-            getNameByValue(syso_hh_types, input$syso_hh_type),
-            " Households"
-          ),
+        str_detect(
```

Comment:
@kiadso won't this ALWAYS be false because none of the HH types contain the word "Household" anymore?

Reply:
yes, I just never know if they're going to change the labels again later, but yes for now

Reply:
I'd rather keep the code clean and reflecting the current decisions. If we change the labels, it should be pretty obvious  that the Total line looks weird. We could also always add some sort of unit test if you're worried we might miss that.